### PR TITLE
Storage get count

### PR DESCRIPTION
**Pull request by tessykc**

### CODE REVIEW

File: api/v1/views/__init__.py [hunk 1]
```diff
@@ -4,7 +4,9 @@
 """
 from flask import Blueprint
 
-app_viewa = Blueprint('app_views', __name__, url_prefix='/api/v1')
 
-"""wildcard import from index.py"""
```
Comment (AnnKamau) [inline api/v1/views/__init__.py]: Reviwed

**Review by AnnKamau — COMMENTED** [review]
Need to remove errors

**Review by AnnKamau — CHANGES_REQUESTED** [review]
Work on errors

**Review by tessykc — COMMENTED** [review]
Changes

**Review by AnnKamau — APPROVED** [review]

**Pull request merged**